.github/workflows/cancelling.yml

```diff
@@ -0,0 +1,20 @@
+name: cancel-builds-on-update
+on:
+  workflow_run:
+    workflows: ['github-Linux', 'github-OSX']
+    types: ['requested']
+
+jobs:
+  cancel-duplicate-workflow-runs:
+    name: "Cancel duplicate workflow runs"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: potiuk/cancel-workflow-runs@master
+        name: "Cancel duplicate workflow runs"
+        with:
+          cancelMode: duplicates
+          cancelFutureDuplicates: true
+          token: ${{ secrets.GITHUB_TOKEN }}
+          sourceRunId: ${{ github.event.workflow_run.id }}
+          notifyPRCancel: true
```

Comment:
Not sure if this will trigger 100s of emails.

Reply:
So far I haven't gotten 100s of emails.